Unit-01/07-sql-alchemy-2/templates/Messages/index.html

```diff
@@ -0,0 +1,20 @@
+{% extends 'base.html' %}
+
+{% block content %}
+	<h1>See all messages for {{user.first_name}}</h1>
+	<a href="{{url_for('messages_new', user_id=user.id)}}">Send {{user.first_name}} a message</a>
+	<br>
```

Comment:
FYI, if you find yourself using a lot of `br` tags in your HTML, that's usually a sign you need some CSS. HTML should be focused on content.